R/build-reference-index.R

```diff
@@ -104,6 +104,9 @@ check_missing_topics <- function(rows, pkg) {
 
   missing <- !in_index & !pkg$topics$internal
   if (any(missing)) {
+    if (tolower(Sys.getenv("CI", "false")) == "true") {
+      stop(c("Topics missing from index: ", unname(pkg$topics$name[missing])))
+    }
```

Comment:
I think the logic is clearer if you followed this with an else branch.

Reply:
Done in 024ac28